### PR TITLE
Create islandora-8-multilingual.md

### PR DESCRIPTION
**Pull request by Natkeeran**
**GitHub Issue**: (link)
* https://github.com/Islandora-CLAW/CLAW/issues/939

# What does this Pull Request do?
This PR provides user documentation to setup a basic multilingual Islandora 8 site.

# How should this be tested?
Please go through the steps to setup a multilingual in vagrant.  

# Interested parties
@Islandora-CLAW/committers

### CODE REVIEW

File: docs/user-documentation/islandora-8-multilingual.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+You can place the default language selector block to switch between languages. To create the language switcher block go to Structure >> Block layout. Click Place block in a region of your choice.  Search for `Language switcher` block and click `Place block`.
+
+## Adding Multilingual Menu
+From the top menu, go to Configuration >> Regional and language >> Content language and translation. Check `Custom menu link` under `Custom language settings`. Scroll down to `Custom menu link` section and check all the relevant fields and Save the configurations. Clear the cache.  
+
+From the top menu, go to Structure >> Menu. Edit "Main navigation" menu. Default home menu item cannot be translated due to [this issue](https://www.drupal.org/project/drupal/issues/2838106). Disable that menu item. Click `Add link` to create a new menu item. Provide a menu title (i.e Home) and input `<front>` for the link field. Save. Right click on the Operations beside the new menu link and click the Translate button. Translate the menu link title for the language added above and save.
```
Comment (manez) [inline docs/user-documentation/islandora-8-multilingual.md]: The section is titled Menus in Drupal, not Menu

File: docs/user-documentation/islandora-8-multilingual.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Drupal 8 comes with four modules for [multilingual support](https://www.drupal.org/docs/8/multilingual/choosing-and-installing-multilingual-modules). Islandora 8 enables Language and Content Translation modules by default.
+
+## Adding Languages
+From the top menu, go to Configuration >> Regional and language >> Languages (`http://localhost:8000/admin/config/regional/language`). Add a language. 
+
+You can place the default language selector block to switch between languages. To create the language switcher block go to Structure >> Block layout. Click Place block in a region of your choice.  Search for `Language switcher` block and click `Place block`.
+
+## Adding Multilingual Menu
+From the top menu, go to Configuration >> Regional and language >> Content language and translation. Check `Custom menu link` under `Custom language settings`. Scroll down to `Custom menu link` section and check all the relevant fields and Save the configurations. Clear the cache.
```
Comment (manez) [inline docs/user-documentation/islandora-8-multilingual.md]: Add (Configuration >> Development >> Performance) after "Clear the cache"

**Comment by rosiel** [comment]
Looks great!
- Can you make a link to this page in https://github.com/Islandora-CLAW/CLAW/blob/master/mkdocs.yml so it'll show up on https://islandora-claw.github.io/CLAW ? 
- Can you fix "repositiry" (twice) near the end of the doc?

**Comment by Natkeeran** [comment]
@rosiel made the requested changes.

**Pull request merged**